### PR TITLE
FMWK-265 Prepare release 4.6.0

### PR DESCRIPTION
**Pull request by agrgr**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,20 +26,20 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <springdata.commons>3.1.2</springdata.commons>
         <springdata.keyvalue>3.1.2</springdata.keyvalue>
-        <spring-cloud-starter-bootstrap>4.0.3</spring-cloud-starter-bootstrap>
+        <spring-cloud-starter-bootstrap>4.0.4</spring-cloud-starter-bootstrap>
         <spring-boot-starter-test>3.1.0</spring-boot-starter-test>
```
Comment (roimenashe) [inline pom.xml:30]: There is new releases available for `spring-boot-start-test`, I assume 3.2.0 is for the new Spring Boot 3.2 but we can still bump it to 3.1.6 (latest of 3.1.x) if we keep Spring Boot at 3.1

Comment (roimenashe) [reply pom.xml:30]: Same thing for `commons` and `keyvalue`? bump to 3.1.6 if possible and if we are not waiting for 3.2.

File: pom.xml [hunk 2]
```diff
@@ -26,20 +26,20 @@
...
         <maven.javadoc.plugin>3.3.0</maven.javadoc.plugin>
         <maven.gpg.plugin>1.6</maven.gpg.plugin>
-        <aerospike-client>7.1.0</aerospike-client>
-        <aerospike-reactor-client>7.0.0</aerospike-reactor-client>
+        <aerospike-client>7.2.0</aerospike-client>
+        <aerospike-reactor-client>7.1.0</aerospike-reactor-client>
         <reactor-test>3.5.7</reactor-test>
```
Comment (roimenashe) [inline pom.xml:35]: Latest is 3.6.0 but there is also 3.5.12 if 3.6 presents a breaking change that we don't want to include

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       # Checkout repo using https://github.com/marketplace/actions/checkout
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up JDK 17
         uses: actions/setup-java@v3
```
Comment (reugn) [inline .github/workflows/build.yml:18]: v4 is available

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Import GPG Key
         uses: crazy-max/ghaction-import-gpg@v5
```
Comment (reugn) [inline .github/workflows/release.yml:11]: v6 is available

**Review by roimenashe — APPROVED** [review]

**Pull request merged**